src/App.jsx

```diff
@@ -1,10 +1,20 @@
 import React from "react";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
+import Header from "./components/Header";
+import Home from "./components/Home";
+import Events from "./components/Events";
+import About from "./components/About";
 
 const App = () => {
   return (
-    <div>
-      <h1>Hello fika world!</h1>
-    </div>
+    <BrowserRouter>
+      <Routes>
+        <Route exact path="/" element={<Home />} />
+        <Route path="/events" element={<Events />} />
+        <Route path="/about" element={<About />} />
```

Comment:
Could we set Events as the home page, so that when we land on the site the first thing we see is the list of Fika? And then have another one called Departments.